packages/atomic/src/components/common/interface/store.ts

```diff
@@ -1,5 +1,6 @@
 import {createStore} from '@stencil/store';
 import {isInDocument} from '../../../utils/utils';
+import {ResultListInfo} from '../../search/atomic-search-interface/store';
```

Comment:
`ResultListInfo` is defined in the same file as the Search store, I think it should be moved here.